partcad/src/partcad/test/cad.py

```
@@ -0,0 +1,24 @@
+# Author: Roman Kuzmenko
+# Created: 2025-01-03
+#
+# Licensed under Apache License, Version 2.0.
+#
+
+from .test import Test
+from .. import logging as pc_logging
+
+
+class CadTest(Test):
+    def __init__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)